javascript/chronometer.js

```diff
@@ -1,26 +1,33 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval( () => this.currentTime++, 1000)
   }
   getMinutes() {
-    // ... your code goes here
+    let minutes = Math.floor(this.currentTime / 60);
+    return minutes
   }
   getSeconds() {
-    // ... your code goes here
+    let minutes = Math.floor(this.currentTime / 60);
+    let seconds = this.currentTime - minutes * 60;
+    return seconds
   }
```

Comment:
Good work Maya! You can make your solution even easier if you use a modulo operator % if you're unfamiliar with it, check out [this link](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math#Arithmetic_operators)
```
return this.currentTime % 60;